### PR TITLE
add note on de-duping styled-components

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Resolves N/A

R&L have run into the same issue we had in #115. Let's document the fix.

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by sareh** [comment]
Looks like there are other recommended fixes, rather than our quite crude method of removing the duplicated `styled-components` from `node_modules`: https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules

**Review by sareh — DISMISSED** [review]

**Comment by ChrisBAshton** [comment]
Fair point. Ben & I looking at this in the office. Will update my README instructions accordingly.

File: README.md [hunk 1]
```diff
@@ -125,6 +125,8 @@ Our recommended `.babelrc` config for this is here:
 
 [See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)
 
+**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicated `styled-components` somewhere in your dependencies, which is not compatible with `styled-components` which acts as a singleton. You can run [npm dedupe](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) or [write your own bash script](https://github.com/BBC-News/psammead/blob/9f30156d5a602b0dbeb427c29568022284746fee/package.json#L9) to resolve the duplication.
```
Comment (ChrisBAshton) [inline README.md]:
```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages, which conflicts with its singleton structure. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups.
```

**Comment by sareh** [comment]
@ChrisBAshton It'd be good to suggest a first step of updating the `styled-components` version in the consuming application to be the same as that of these psammead components. This seemed to work for the R+L team's usecase.

File: README.md [hunk 2]
```diff
@@ -125,6 +125,8 @@ Our recommended `.babelrc` config for this is here:
 
 [See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)
 
+**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages, which conflicts with its singleton structure. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups.
```
Comment (ChrisBAshton) [inline README.md]:
```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages, which conflicts with its singleton structure. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups, although there is an [issue with symlinking `jest-styled-components` mangling snapshots](https://github.com/styled-components/jest-styled-components/issues/218).
```

File: README.md [hunk 3]
```diff
@@ -125,6 +125,8 @@ Our recommended `.babelrc` config for this is here:
 
 [See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)
 
+**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages, which conflicts with its singleton structure. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups, although there is an [issue with symlinking `jest-styled-components` mangling snapshots](https://github.com/styled-components/jest-styled-components/issues/218).
```
Comment (ChrisBAshton) [inline README.md]:
```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups (although there is an [open issue with symlinking `jest-styled-components`](https://github.com/styled-components/jest-styled-components/issues/218)). Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.
```

File: README.md [hunk 4]
```diff
@@ -125,6 +125,8 @@ Our recommended `.babelrc` config for this is here:
 
 [See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)
 
+**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups (although there is an [open issue with symlinking `jest-styled-components`](https://github.com/styled-components/jest-styled-components/issues/218)). Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.
```
Comment (ChrisBAshton) [inline README.md]:
```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups. Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.
```

**Comment by ChrisBAshton** [comment]
`--hoist` works (apologies, jest-styled-components! https://github.com/styled-components/jest-styled-components/issues/218)

Ben is raising a PR to replace our hack.

Have raised a corresponding issue: https://github.com/BBC-News/psammead/issues/240

**Review by sareh — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -125,6 +125,8 @@ Our recommended `.babelrc` config for this is here:
 
 [See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)
 
+**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups. Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.
```
Comment (bcmn) [inline README.md]:
```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups such as Psammead's. Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.
```

**Pull request merged**